### PR TITLE
Ruby 2.5 / Ubuntu 18.04/19 Changes

### LINKED ISSUE #22: Update to Ubuntu 18.04 LTS
It'll be released on 2018-04-26 and it'd be nice to update Lobsters to move up to [Ruby 2.5.1](https://packages.ubuntu.com/search?keywords=ruby&searchon=names&suite=bionic&section=all).

Because this is a big update, we should use the `lobsters-test` staging server to confirm the playbook can configure and start Lobsters from scratch. Our current install included some hand config, so we can build confidence that all of that config has been encoded into ansible properly.

### PR DESCRIPTION
**Pull request by hmadison**
Resolves #22

I've also gone in and resolved the deprecation warnings given off by Ansible 2.8 and moved `config/database.yml` into Ansible to make setting up the test server after a wipe a bit simpler.

I'm currently running master [on the test server](http://lobsters-test.xen.prgmr.com/)

### CODE REVIEW

File: roles/lobsters/templates/database.yml.j2 [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{{ env }}:
+  adapter: mysql2
+  encoding: utf8mb4
+  database: lobsters_{{ env }}
+  pool: 5
+  username: lobsters
+  password:
+  socket: /var/run/mysqld/mysqld.sock
```
[the remark this answers is not in the document]
Comment (hmadison) [reply roles/lobsters/templates/database.yml.j2]: Updated. Thanks!

**Comment by thomasdziedzic** [comment]
Seems like you introduced some kind of conflicts into the repo:

```
Conflicting files
roles/lobsters/tasks/main.yml
```

File: roles/lobsters/vars/main.yml [hunk 2]
```diff
@@ -10,6 +10,6 @@ required_packages:
   - make
   - mariadb-client
   - nodejs
-  - ruby2.3
-  - ruby2.3-dev
+  - ruby2.5
+  - ruby2.5-dev
```
Comment (alanpost) [inline roles/lobsters/vars/main.yml]: This commit I can't put in until production is running Ubuntu 18.  May I ask it to be submitted as a separate PR?  The rest of this work is ansible-related, and I can put in prior to upgrading production.

**Comment by alanpost** [comment]
I just fiddled with the PR/merge settings in this repository, forbidding merge commits to master.  @hmadison, will you force push these changes to remove the merge commit that appears here?

**Comment by pushcx** [comment]
I'm curious, why did you forbid merge commits to master? To better represent that this is coupled to a single running system that can't have branches?

**Comment by alanpost** [comment]
> I'm curious, why did you forbid merge commits to master? To better represent that this is coupled to a single running system that can't have branches?

It's really nothing more than habit and preference.  I like rebase workflows when I can get them (i.e., multi-branch codebases that don't need or benefit from octopus merging).  In truth, across all the projects I'm involved in, I see and work with both.

File: roles/lobsters/vars/main.yml (hunk 2 above)
Comment (hmadison) [reply roles/lobsters/vars/main.yml]: Done.

**Pull request merged**